tests/Postman/collection.json

```diff
@@ -1,2021 +1,2279 @@
-                  ]
-                }
-              },
-              "response": []
-            },
-            {
-              "name": "SW: create price grp",
-              "event": [
-                {
-                  "listen": "test",
-                  "script": {
-                    "exec": [
-                      "pm.test(\"Status code is 201\", function () {\r",
-                      "    pm.response.to.have.status(201);\r",
-                      "});\r",
-                      "\r",
-                      "pm.test(\"Get id of created product\", function () {\r",
-                      "    var jsonData = pm.response.json();\r",
-                      "    pm.expect(jsonData.data.id).to.be.a(\"number\");\r",
-                      "    pm.environment.set('created_product2_id', jsonData.data.id);\r",
-                      "});"
-                    ],
-                    "type": "text/javascript"
-                  }
-                }
-              ],
-              "request": {
-                "method": "POST",
-                "header": [],
-                "body": {
-                  "mode": "raw",
-                  "raw": "{\r\n    \"name\": \"API: price grp product\",\r\n    \"active\": true,\r\n    \"tax\": 19,\r\n    \"supplier\": \"Active Runners LLC\",\r\n    \"categories\": [\r\n        {\r\n            \"id\": {{gen_category_main_id}}\r\n        }\r\n    ],\r\n    \"lastStock\": true, // stock cannot go into negatives (saleable)\r\n    \"priceGroupActive\": true,\r\n    \"priceGroupId\": 500,\r\n    \"mainDetail\": {\r\n        \"number\": \"API-PRICE-GRP\",\r\n        \"active\": true,\r\n        \"prices\": [\r\n            {\r\n                \"customerGroupKey\": \"EK\",\r\n                \"from\": \"1\",\r\n                \"to\": \"3\",\r\n                \"price\": 200,\r\n                \"pseudoPrice\": 250, // fake price to cross out\r\n                \"basePrice\": 90\r\n            },\r\n            {\r\n                \"customerGroupKey\": \"EK\",\r\n                \"from\": \"4\",\r\n                \"to\": \"7\",\r\n                \"price\": 100,\r\n                \"pseudoPrice\": 250,\r\n                \"basePrice\": 90\r\n            }\r\n        ],\r\n        \"inStock\": 500,\r\n        \"minPurchase\": 2,\r\n        \"maxPurchase\": 10,\r\n        \"purchaseSteps\": 2,\r\n        \"position\": 5\r\n    }\r\n}",
-                  "options": {
-                    "raw": {
-                      "language": "json"
-                    }
-                  }
-                },
-                "url": {
-                  "raw": "{{sw_endpoint_api}}/articles",
-                  "host": [
-                    "{{sw_endpoint_api}}"
-                  ],
-                  "path": [
-                    "articles"
-                  ]
-                }
-              },
-              "response": []
-            }
-          ]
-        }
-      ]
+    "info": {
```

Comment:
2 spaces plz 🙈

Reply:
I could, but um, I am just following the SW5 .editorconfig policy: https://github.com/shopware/shopware/blob/trunk/.editorconfig#L11

I could overwrite to a 2 if that's what you wish, submit a file with our repo with just:
```
[*.json]
indent_size = 2
```
What you think @Rekhyt ?

Reply:
I think it says 2 spaces in the file you linked: https://github.com/shopware/shopware/blob/trunk/.editorconfig#L19

Plus, it was 2 before, so the diff is all over the place now. Not unsovable but if we don't have to switch to 4 we shouldn't.

Reply:
Ye, my bad, wrong repo (that's sw6), was looking at local config & made a mistake in repo link https://github.com/shopware5/shopware/blob/5.7/.editorconfig#L11

No problem though, I created a config rewrite.

Reply:
That's cool, thank you 👍